app/ModelSerializers/Summit/ISummitEventSerializerTypes.php

```diff
@@ -0,0 +1,22 @@
+<?php namespace App\ModelSerializers\Summit;
+/**
+ * Copyright 2024 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+/**
+ * Interface ISummitEventSerializerTypes
+ * @package ModelSerializers
+ */
+interface ISummitEventSerializerTypes
+{
+    const OverflowStream = 'OVERFLOW_STREAM';
```

Comment:
move this to IPresentationSerializerTypes

Reply:
done